Dockerfile

```diff
@@ -3,8 +3,11 @@ FROM alpine:latest
 ENV S6_URL https://github.com/just-containers/s6-overlay/releases/download/v2.0.0.1/s6-overlay-amd64.tar.gz
 ENV S6_SHA256SUM df235428444be7737caf363e5dcfb58e3022fa611f5a27fe309ecd6fc1755cda
 
+# Set sane defaults for the application
 ENV APP_ENV production
 ENV APP_DEBUG false
+
+# Include our app directory into $PATH
```

Comment:
You don't need to comment everything. Some things are self explanatory like this.